phpstan.neon

```diff
@@ -9,5 +9,6 @@ parameters:
         - '#Method ([\w\\\:_\(\)]+) has parameter ([\$\w]+) with a nullable type declaration.#'
         - '#Method ([\w\\\:_\(\)]+) has parameter ([\$\w]+) with null as default value.#'
         - '#Method ([\w\\\:_\(\)]+) has a nullable return type declaration.#'
+        - '#Dynamic call to static method ([\w\\\:\(\)\<\>]+).#'
```

Comment:
This should only apply to the eloquent namespace as discussed in slack because this will ignore everything even if it isn't related to eloquent.

Reply:
@faustbrian what do you think of this regex, too strict?